user/enterprise/trusty-beta.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Trusty containers for Enterprise (beta)
+layout: en_enterprise
+
+---
+
+**Please note: This is a beta feature. We would definitely like to hear if you run into any problems with the Trusty containers. If you have any questions or run into problems, please write an email to: [enterprise@travis-ci.com](mailto:enterprise@travis-ci.com).**
+
+First of all you need to be running Travis Enterprise 2.1.9 or higher.
+
+1. SSH into the platform machine
+2. Run `travis console`
+3. Then run `Travis::Features.enable_for_all(:template_selection); Travis::Features.enable_for_all(:multi_os)`
+4. You can now exit the console and disconnect from the Travis Enterprise platform machine
+
+**Please be aware that the commands below need to be run on fresh machines. The worker machine(s) with the Precise (Ubuntu 12.04) images can co-exist next to the Trusty workers.**
```

Comment:
Still sounds a bit like they can co-exist on the same machine. Maybe: 

"While worker machines with the older Precise (Ubuntu 12.04) containers can co-exist with Trusty worker machines, you cannot have a worker with both Precise and Trusty containers on the same machine. They must be setup on separate fresh machines."

Feels too long there, still. Maybe I'm just muddying it up completely? Thoughts?